doc/api/cluster.md

```diff
@@ -374,24 +395,30 @@ if (cluster.isMaster) {
 
 ### worker.suicide
 
-* {Boolean}
+    Stability: 0 - Deprecated: Use [`worker.exitedVoluntarily`][] instead.
 
-Set by calling `.kill()` or `.disconnect()`, until then it is `undefined`.
+An alias to [`worker.exitedVoluntarily`][].
 
-The boolean `worker.suicide` lets you distinguish between voluntary and accidental
-exit, the master may choose not to respawn a worker based on this value.
+Set by calling `.kill()` or `.disconnect()`. Until then, it is `undefined`.
+
+The boolean `worker.suicide` lets you distinguish between voluntary
+and accidental exit, the master may choose not to respawn a worker based on
+this value.
```

Comment:
Not sure if there is a reason to duplicate the `exitedVoluntarily` documentation here, just a link should do it IMO.
I don't have a strong opinion on this, though. It would perhaps be better to wait for more opinions here.